products/amazon-documentdb.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Amazon DocumentDB
+category: service
+tags: amazon database
+iconSlug: amazondocumentdb
+permalink: /amazon-documentdb
+releasePolicyLink: https://docs.aws.amazon.com/documentdb/latest/developerguide/compatibility.html
+releaseDateColumn: true
+releaseColumn: true
+
+releases:
+-   releaseCycle: "5.0"
+    releaseDate: 2023-03-01
+    eol: false
+    eoes: false
+    latest: "5.0"
+    latestReleaseDate: 2023-03-01
+
+-   releaseCycle: "4.0"
+    releaseDate: 2020-11-09
+    latest: "4.0"
+    eol: false
+    eoes: false
+    latestReleaseDate: 2020-11-09
+
+-   releaseCycle: "3.6"
+    releaseDate: 2019-01-09
+    eol: false
+    eoes: false
+    latest: "3.6"
+    latestReleaseDate: 2019-01-09
+
+---
+
+> [Amazon DocumentDB (with MongoDB compatibility)](https://aws.amazon.com/documentdb/) is a fast, reliable, and fully 
+> managed database service that makes it easy for you to set up, operate, and scale MongoDB-compatible databases.
+
+The DocumentDB engine is released approximately once a month for maintenance version. Major version support for [MangoDB](/mongodb) versions occurs approximately very two years. 
+
```

Comment:
I did not find any information about how much time DocumentDB releases are supported. Do you know they are EOL dates or at least a list of supported releases documented somewhere ?